Makefile

```diff
@@ -0,0 +1,9 @@
+.phony: build
+build:
+	PKG_CONFIG_PATH=/usr/local/share/pkgconfig
```

Comment:
Why is this needed?

Reply:
The go compiler uses pkg-config to link the go binary to static C libs. Hence, the path to pkg-config files must set as an environment variable.